packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx

```diff
@@ -0,0 +1,456 @@
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} hideFeedback />
+
+<ArticleContent>
+
+## 📣 What's new in Paste
+
+### ALL the new components!
+
+#### Switch
+
+![](/images/gifs-2022-09-23-paste-newsletter/switch-example.gif)
+
+The Switch component makes interacting with binary states a breeze.
+Our Switch incorporates animation and icons to make its pressed/unpressed state visually clear, while its built-in
+accessibility features communicate the state clearly to users. Take it for a spin!
+
+<Paragraph>
+  <Anchor href="/components/switch">Check out the Switch docs</Anchor>
+</Paragraph>
+
+#### Inline Code
+
+![](../../assets/images/articles/2022-09-23-paste-newsletter/inline-code-example.png)
+
+If your snippet of code is small and inline with the rest of your text,
+Inline Code is the way to go. Its contents stand out with a subtle background and monospace font.
+Use it for code, ID&apos;s, and keys.
+
+<Paragraph>
+  <Anchor href="/components/inline-code">Check out the Inline Code docs</Anchor>
+</Paragraph>
+
+#### Code Block
+
+![](/images/gifs-2022-09-23-paste-newsletter/code-block-example.gif)
+
+Presenting Code Block! This component allows you to display code with syntax highlighting for 13 programming languages.
+It features tabs to show different blocks of code in the same area, a copy code button, and an optional external link button leading to the source code.
+
+<Paragraph>
+  <Anchor href="/components/code-block">Check out the Code Block docs</Anchor>
+</Paragraph>
+
+#### Callout
+
+![](../../assets/images/articles/2022-09-23-paste-newsletter/callout-example.png)
+
+With five different variants to choose from, all your imperative, static information will be
+hard to miss inside of the Callout component! Don&apos;t forget to set its heading level according
+to the outline of your page for the best user experience.
+
+<Paragraph>
+  <Anchor href="/components/callout">Check out the Callout docs</Anchor>
+</Paragraph>
+
+#### Minimizable Dialog
+
+![](/images/gifs-2022-09-23-paste-newsletter/minimizable-dialog-example.gif)
+
+As part of our Conversations UI kit, we&apos;ve added the Minimizable Dialog component.
+Its power lies in its flexibility; use it with a static button, a floating button, or use its included state hook
+for full control of its internal state. Making a chat log has never been easier with Paste.
+
+<Paragraph>
+  <Anchor href="/components/minimizable-dialog">Check out the Minimizable Dialog docs</Anchor>
+</Paragraph>
+
+#### 42 new icons, including a full suite of contact center icons!
+
+![](../../assets/images/articles/2022-09-23-paste-newsletter/new-icons-example.png)
+
+We&apos;ve got new icons! A full suite of contact center icons and more are now available.
+Thanks to <strong>Fazle Jahangir, Isabela Razetti,</strong> and <strong>Porlin Chin</strong> for their close collaboration on these icons.
+
+<br />
```

Comment:
todo: remove this br

Reply:
@serifluous I see you, but I'd like to push back on doing this in this PR. It's a layout thing as opposed to something wrong with the content, and we've delayed the newsletter rollout already from the email restrictions. I'd rather fix this is another PR on the queue.